react/src/components/card/SprkCard.js

```diff
@@ -1,55 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
-import SprkCardHighlightedHeader from './components/SprkCardHighlightedHeader/SprkCardHighlightedHeader';
-import SprkCardTeaser from './components/SprkCardTeaser/SprkCardTeaser';
 
 const SprkCard = (props) => {
-  const {
-    additionalClasses,
-    additionalContentClasses,
-    children,
-    highlightedHeaderConfig,
-    idString,
-    isStandout,
-    teaserConfig,
-    variant,
-  } = props;
-
-  const GetCardContent = () => {
-    if (variant === 'highlightedHeader') {
-      return (
-        <SprkCardHighlightedHeader
-          highlightedHeaderConfig={highlightedHeaderConfig}
-        />
-      );
-    }
-
-    if (variant === 'teaser') {
-      return <SprkCardTeaser teaserConfig={teaserConfig} />;
-    }
-
-    return (
-      <div
-        className={cx(
-          'sprk-c-Card__content sprk-o-Stack__item',
-          additionalContentClasses,
-        )}
-      >
-        {children}
-      </div>
-    );
-  };
+  const { additionalClasses, children, idString, isStandout, ...other } = props;
 
   return (
     <div
       className={cx('sprk-c-Card sprk-o-Stack', additionalClasses, {
         'sprk-c-Card--standout': isStandout,
-        'sprk-u-TextAlign--center': teaserConfig.media.mediaVariant === 'icon',
       })}
       data-id={idString}
+      {...other}
```

Comment:
We weren't spreading the other props before so I added that in as a new feature.